zstacklib/zstacklib/utils/shell.py

```
@@ -46,8 +46,13 @@ def __init__(self, cmd, workdir=None, pipe=True):
         self.stderr = None
         self.return_code = None
 
-    def raise_error(self):
-        err = []
+    def raise_error(self, err=()):
+        """
+        :param err: extend error message
+        :return:
+        """
+        if err is ():
```

Ruff (0.8.0)

Use `==` to compare constant literals

Replace `is` with `==`

(F632)